project1.html

```diff
@@ -110,11 +110,35 @@
     <a class="Vector2"><img src="Vector2.png" alt=""></a>
     <a class="Vector3"><img src="Vector2.png" alt=""></a>
     <div><h4 class="Frameworks">Frameworks</h4></div>
-  <a class="Separator1"><img src="Separator Bottom.jpg" alt=""></a></div>
+  <a class="Separator1"><img src="SeparatorBottom.jpg" alt=""></a></div>
   <div><h4 class="Skills">Skills</h4></div>
-    <a class="Separator2"><img src="Separator Bottom.jpg" alt=""></a>
-    <a class="Separator1"><img src="Separator Bottom.jpg" alt=""></a>
+    <a class="Separator2"><img src="SeparatorBottom.jpg" alt=""></a>
+    <a class="Separator1"><img src="SeparatorBottom.jpg" alt=""></a>
 </div>    
+
+<div class="add-contact add-contact-bckgrnd">
+  <div class="image"></div>>
+
+  <h1>Contact me</h1>
+  <p>If you have an application you are interested in developing, 
+    a feature that you need built or a project that needs coding. 
+    I’d love to help with it </p>
+
+    <form action="https://formspree.io/f/form_id" method="post"  class= "form">
```

Comment:
- [ ] Kindly reconfigure your action to ensure your form is linked to an actual formspree account, currently, data is not submitted on clicking the get in touch button.